### DISCUSSION TITLE
Question with raw message storage in database

### DISCUSSION THREAD

**Discussion by pplulee** [postalserver/postal#2123] [Help with using Postal]
Is there any way to set a size limitation of raw messages that are stored in the database? Or is it possible to change auto delete days?
(the table in a name like taw-2022-xx-xx in each server)
I have tried to change the setting in "Retention" but it seems not working. For example, I set the "Number of days that raw message data will be stored" to 7 days, but there is still older table in the database. The volume size limitation is also not working.

![image](https://user-images.githubusercontent.com/46434255/185787109-229acf02-e87d-44a5-9ef9-533d4c764bdc.png)
![image](https://user-images.githubusercontent.com/46434255/185787125-449b335a-b5f0-4555-87f2-0d77624f54bf.png)

**Comment by willpower232** [comment]
Retention settings can be edited in the Advanced Settings tab

![image](https://user-images.githubusercontent.com/1619102/185946467-eefead1b-2d88-4f29-8838-7c2521d3b8c6.png)

I believe that the messages are only cleaned up once a day so changing the settings won't take instant effect but will do by this time tomorrow.

**Reply by pplulee** [reply]
Is there any way to change the default setting? So that every new server will use that parameter rather than the default setting.

**Reply by willpower232** [reply]
I do not believe so

https://github.com/postalserver/postal/blob/6bace2c9053e8bdfc2689c3c92a9fc066d124401/app/models/server.rb#L69-L71

**Reply by pplulee** [reply]
The clean-up is still not working for me. The setting looks like this
![image](https://user-images.githubusercontent.com/46434255/186635222-a5fb557e-2f88-44b1-a173-3ec6c5b86d91.png)
Actually, in the above situation, I have already waited for several days. I saw it didn't work so I came to ask

**Reply by willpower232** [reply]
Can you confirm with `postal status` that you have a cron process running?

**Reply by pplulee** [reply]
Yes, it is running. It worked after I restarted the server. Thank you for replying!
